pydis_site/apps/content/resources/guides/python-guides/setting-different-statuses-on-your-bot.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Setting Different Statuses to Set Your Bot
+description: How to personalize your Discord bot status
```

Comment:
```suggestion
description: How to personalize your discord bot status
```
A *very* minor nitpick. You don't need to accept this if you don't want, just figured I'd point it out haha.

Reply:
Thanks for the suggestion, but I don't think lowercasing Discord would be in line with the casing in other guides.

For example:

- https://www.pythondiscord.com/pages/guides/python-guides/discordpy/
- https://www.pythondiscord.com/pages/guides/python-guides/discord-embed-limits/

What do you think?

Reply:
It should be uppercased because it is referring to Discord (the app/company) and not "discord" the word.

Reply:
> Thanks for the suggestion, but I don't think lowercasing Discord would be in line with the casing in other guides.
> 
> 
> 
> For example:
> 
> 
> 
> - https://www.pythondiscord.com/pages/guides/python-guides/discordpy/
> 
> - https://www.pythondiscord.com/pages/guides/python-guides/discord-embed-limits/
> 
> 
> 
> What do you think?

That makes sense, I didn't think of that haha